src/main/java/thePackmaster/powers/dimensiongate2pack/SchemePower.java

```diff
@@ -0,0 +1,69 @@
+package thePackmaster.powers.dimensiongate2pack;
+
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SchemePower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("SchemePower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String DESCRIPTIONS[] = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public SchemePower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, true, owner, amount);
+
+    }
+
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (!card.purgeOnUse && this.amount > 0 && (Wiz.getLogicalCardCost(card) == 0 || Wiz.getLogicalCardCost(card) == 1)) {
```

Comment:
Would it make more sense to have this stack by affecting the first X cards each turn? That's how the base game effects for "the first/next time" stack. I'm mostly thinking of Echo Form and other duplication effects (you don't play the first card X times, you play the first card X cards twice), but this is also true for things like the double damage effect from Phantasmal Killer (you don't get triple damage, you get double damage for more turns).

That also limits degenerate cases, e.g. when you have Apparitions. Playing an Apparition twice is already very powerful, and anything that lets you play it 3+ times is very dangerous. If this affected the first X cards, at least after doubling your Apparition you now have to double other, less powerful cards for your next plays.

I feel that the base game approach to stacking powers like this is a good precedent to follow, both to match the expectation people might have and for the balance reasons I described.

Reply:
Yeah, that makes sense.  Echo Form has set a precedent.  Will do.